canonical-websites/www.ubuntu.com
# Clean up redirects.yaml

## Done

* Use more aggressive regex to simplify redirects.yaml
* Removed a lot of very old redirects that don't seem worth preserving
* Fixed some redirects (namely Kylin)
* Added 'backed-by-canonical' to deleted pages
* Created a test-redirect script that tests redirects and known good URLs to make sure regressions are ok
* Provided a google sheet to test and look at script results

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and click around to see that all links currently still work
- test urls from the redirects to see that they redirect (note, some are simplified)
- use this [google sheet](https://docs.google.com/spreadsheets/d/1mu-VtLnuOw8Y-V9KvxI7kYYgMRBiWLLlLSeGwwbkNK0/edit#gid=2014315232) to test and look at script results
